app/src/main/java/com/teamagam/gimelgimel/app/common/utils/UsernameGenerator.java

```diff
@@ -0,0 +1,35 @@
+package com.teamagam.gimelgimel.app.common.utils;
+
+import android.content.Context;
+import android.content.res.Resources;
+
+import com.teamagam.gimelgimel.R;
+
+import java.util.Random;
+
+public class UsernameGenerator {
+
+    private static final String SPACE = " ";
+    private static final Random rand = new Random();
+
+    public static String generateUsername(Context context) {
+        Resources res = context.getResources();
+        String[] animals = res.getStringArray(R.array.name_generator_animals);
+        String[] adjectives = res.getStringArray(R.array.name_generator_adjectives);
+        return capitalizeFirstLetters(adjectives[rand.nextInt(adjectives.length)]) + SPACE +
```

Comment:
And wouldn't it be prettier if it was a virtual method of an instance object that initializes it's source arrays at the constructor / gets them as parameters and the generateUsername would only have 
1. pick random adjective
2. pick random animal
3. merge them
4. return capitalized string

?